templates/order_getitems.html.twig

```diff
@@ -0,0 +1,398 @@
+{#
+ # -------------------------------------------------------------------------
+ # Order plugin for GLPI
+ # -------------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of Order.
+ #
+ # Order is free software; you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation; either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # Order is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with Order. If not, see <http://www.gnu.org/licenses/>.
+ # -------------------------------------------------------------------------
+ # @copyright Copyright (C) 2009-2023 by Order plugin team.
+ # @copyright 2015-2025 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ # @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ # @link      https://github.com/pluginsGLPI/order
+ # -------------------------------------------------------------------------
+ #}
+
+{% set datatable_id = datatable_id|default('datatable' ~ random()) %}
+{% set showmassiveactions = canedit %}
+{% set table_visible = table_visible|default(false) %}
+{% if hide_and_show %}
+    <button id="toggle-table-{{ datatable_id }}" class="btn btn-link btn-sm mb-2" type="button">
+        {% if table_visible %}
+            <i class="ti ti-eye-off"></i>
+        {% else %}
+            <i class="ti ti-eye"></i>
+        {% endif %}
```

Comment:
Can you add text `Expand all`/ `Collapse all`

and use

- `ti-sort-descending`
- `ti-sort-ascending`